### PR TITLE
GOB: Detection improvements

### PR DESCRIPTION
**Pull request by BJNFNE**

### CODE REVIEW

File: engines/gob/detection/tables_adi2.h [hunk 1]
```diff
@@ -215,7 +215,7 @@
 		"Non-Interactive Demo",
 		AD_ENTRY4s("demo.scn", "16331b4db31b153f241ebcee49b7383d", 170,
 				"demadi2d.vmd",	"658b5502bf0aeb46a39788f6a07e3d97", 7552841,
-                "demoarchd.vmd", "a93184f0e297bb2e1ebfd64c1038e1ae", 5946788,
+                "demarchd.vmd", "a93184f0e297bb2e1ebfd64c1038e1ae", 5946788,
```
Comment (sev-) [inline engines/gob/detection/tables_adi2.h:218]: So, that means that the previous conversion was not tested?

Do you in general test all changes in your Pull Requests?

Comment (BJNFNE) [reply engines/gob/detection/tables_adi2.h:218]: I test in general, all my changes, This commit fixes now the typo, The Demo is now detected by ScummVM without problems.

Comment (BJNFNE) [reply engines/gob/detection/tables_adi2.h:218]: i have written by accident demo instead of dem

**Comment by sev-** [comment]
Thank you!

**Pull request merged**